indexd/dos/blueprint.py

```diff
@@ -120,22 +127,26 @@ def indexd_to_dos(record):
 
 @blueprint.errorhandler(UserError)
 def handle_user_error(err):
-    return flask.jsonify(error=str(err)), 400
+    ret = {"msg": str(err), "status_code": 400}
```

Comment:
why do you need to return the status code also in the response data?

Reply:
i don't know why it was done this way, but a production website is using a branch with this change so i'd rather not change the contents of the response